feature/amazon/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
     implementation "androidx.annotation:annotation:$annotationVersion"
 
-    latestDependenciesImplementation "com.amazon.device:amazon-appstore-sdk:$amazonVersion"
+    implementation "com.amazon.device:amazon-appstore-sdk:$amazonVersion"
```

Comment:
Why this change?

Reply:
ah, this was leftover from when i thought unity would be using the same version. will revert